scripts/gulpfiles/build_tasks.js

```diff
@@ -337,6 +339,13 @@ function buildDeps(done) {
     'tests/mocha'
   ];
 
+  function filterOutputs(text, keyword) {
+    return text.split('\n')
+        .filter(
+          (line) => line.indexOf(keyword) >= 0)
+        .join('\n');
+  }
```

Comment:
This function extracts lines that contain the specified keyword.
So I think it would be better to rename the function to something like `extractOutputs`.

Reply:
How about an implementation like below?
```javascript
  /**
   * Extracts lines that contain the specified keyword.
   * @param {string} text output text
   * @param {string} keyword extract lines with this keyword
   * @returns {string} modified text
   */
  function extractOutputs(text, keyword) {
    return text.split('\n')
        .filter((line) => line.includes(keyword))
        .join('\n');
  }
```

Reply:
I changed as I suggested.

Reply:
> This function extracts lines that contain the specified keyword. So I think it would be better to rename the function to something like `extractOutputs`.

Oh yes, sorry—you're quite right!

The revised implementation looks great.  Choice of name is OK too.